--- a/libsrc/libbioimg/formats/jp2/bim_jp2_format.cpp
+++ b/libsrc/libbioimg/formats/jp2/bim_jp2_format.cpp
@@ -252,30 +252,6 @@
         par->buffer_icc.resize(par->image->icc_profile_len);
         memcpy(&par->buffer_icc[0], par->image->icc_profile_buf, par->image->icc_profile_len);
     }
-
-    // Read the main header of the codestream, and if necessary the JP2 boxes
-    if (par->image && par->image->cp_comment) {
-        size_t endpos = 0;
-        size_t startpos = 0;
-        const char* comment;
-        while (true) {
-            ++endpos;
-            if (par->image->cp_comment[endpos] == 0 && par->image->cp_comment[endpos + 1] != 0) {
-                // A single zero character followed by a non-zero character indicates the end
-                // of one comment. So here is a break between two concatenated comments:
-                comment = (par->image->cp_comment + startpos);
-                par->comments.push_back(comment);
-                startpos = endpos + 1;
-            } else if (par->image->cp_comment[endpos] == 0 && par->image->cp_comment[endpos + 1] == 0 && par->image->cp_comment[endpos + 2] == 0) {
-                // Three consecutive zero characters indicate the end of the comments list.
-                // So here is the end of all string comments:
-                comment = (par->image->cp_comment + startpos);
-                par->comments.push_back(comment);
-                break;
-            }
-        }
-    }
-
 }
 
 void jp2GetImageInfo(FormatHandle *fmtHndl) {
